about.html

```diff
@@ -0,0 +1,161 @@
+                totam fuga voluptatibus accusamus magnam explicabo excepturi tempora, 
+                quibusdam magni aperiam consequatur!
+            </P>
+            <span class="dark-about">
+                Please contact us via email for any questions about GlobalMuzik Easter Fiester 2022
+            </span>
+            <a href="">raheemabiodun23@gmail.com</a>
+        </div>
+    </section>
+    <section class="section-about main-event">
+        <h2>GlobalMuzik Easter Fiester 2022 Logo</h2>
+        <hr class="line">
+        <p class="about-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis dignissimos 
+            totam fuga voluptatibus accusamus magnam explicabo excepturi tempora, 
+            quibusdam magni aperiam consequatur!
+        </p>
+        <div class="muzik-logo">
+            <div class="music-icon">
+                <h3>globalMUZIK</h3>
+                <p>
+                    Lorem ipsum dolor amet 
+                    consec. 
+                </p>
+            </div>
+            <img src="./images/mlogo.png" alt="company logo">
+        </div>
+    </section>
+
+    <section class="section-about main-event">
+        <hr class="line2">
+        <h2>See The Past GlobalMusik Events</h2>
+        <hr class="line">
+        <p class="about-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis dignissimos 
+            totam fuga voluptatibus accusamus magnam explicabo excepturi tempora, 
+            quibusdam magni aperiam consequatur! 
+        </p>
+        <ul class="previouse-events">
+            <li class="event one">
+                <h3>2022</h3>
+                <p>GlobalMusik Fiester, LosAngeles</p>
+            </li>
+            <li class="event two">
+                <h3>2022</h3>
+                <p>GlobalMusik Fiester, Berlin</p>
+            </li>
+        </ul>
+    </section>
+
+    <main>
+        <section class="partners">
+            <h4>Partners</h4>
+            <hr class="line">
+            <ul class="sponsors">
+                <li><img src="./images/sponsor1.png" alt="addidas-logo"></li>
+                <li><img src="./images/sponsor2.png" alt="flyemirate-logo"></li>
+                <li><img src="./images/sponsor4.jpg" alt="audi-logo"></li>
+                <li><img src="./images/sponsor5.png" alt="nivea-logo"></li>
+                <li><img src="./images/sponsor6.png" alt="nike-logo"></li>
+            </ul>
+        </section>
+        <footer>
+            <ul class="footer">
+                <li class="logo">
+                    <img src="./images/mlogo.png" alt="music-logo" width="100" height="100">
+                </li>
+                <li class="foot">
+                    <h6>2022 the MUSICAL Event. Some Rights Reserved.</h6>
+                    <button name="button" type="submit" class="btn">Credit</button>
+                </li>
+            </ul>
+        </footer>    
```

Comment:
- [ ] As mentioned above in the index.html page, kindly remove the footer from inside the main element as it disrupts the semantic page layout. 
- [ ] The footer for the about page is different from that of the home page, see the images below and kindly make the changes.

_See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/163600398-c142953f-1011-4aeb-9ee6-74cf6a943049.png) | ![image](https://user-images.githubusercontent.com/49621279/163600449-57480dfb-8f30-4e4d-afea-fa58d24e2894.png)